### PR TITLE
Ports pilk and peg nog from Jollystation

### PR DESCRIPTION
**Pull request by Toastgoats**
## About The Pull Request

Exactly what it says on the tin, essentially just ports over https://github.com/Jolly-66/JollyStationCode/pull/110 by CRITAWAKETS 
which has some mood buffs and plenty of fluff, fitting for the official cocktail(s) of this years' toolboxing tournament.

Opening this as a draft since I've never ported code before, probably fucked something up.

## Why It's Good For The Game

Admit it, this was a long time coming.

## Changelog

:cl: Toastgoats and CRITAWAKETS 
add: Pilk and peg nog are here.
imageadd: Ported the pilk and peg nog sprites.
/:cl:

### CODE REVIEW

File: code/datums/mood_events/drink_events.dm [hunk 1]
```diff
@@ -26,3 +26,50 @@
 	description = "<span class='nicegreen'>Amazing taste!</span>\n"
 	mood_change = 50
 	timeout = 10 MINUTES
+	
+/datum/reagent/consumable/pilk
```
Comment (Sealed101) [inline code/datums/mood_events/drink_events.dm:30]: pretty sure you meant to copypaste the mood_events and not reagents here

**Review by Sealed101 — CHANGES_REQUESTED** [review]

File: code/datums/mood_events/drink_events.dm (hunk 1 above)
Comment (Toastgoats) [reply code/datums/mood_events/drink_events.dm:30]: God I'm a fucking moron lmao

File: code/datums/mood_events/drink_events.dm [hunk 2]
```diff
@@ -26,3 +26,13 @@
...
 	mood_change = 50
 	timeout = 10 MINUTES
+	
+/datum/mood_event/full_on_pilk
+	description = "<span class='nicegreen'>I am now full on pilk! That was some amazing bubbly goodness!</span>\n"
+	mood_change = 7
+	timeout = 7 MINUTES
+
+/datum/mood_event/pegged
+	description = "<span class='nicegreen'>OH YEAH, NOW IM PEGGED!</span>\n"
```
Comment (JohnFulpWillard) [inline code/datums/mood_events/drink_events.dm:36]: :face_with_raised_eyebrow:

Comment (Sealed101) [reply code/datums/mood_events/drink_events.dm:36]: image
___
image

**Comment by Toastgoats** [comment]
Agony

**Comment by Toastgoats** [comment]
Well I was about to fix it but the original coder let me know that he doesn't want this specific pr ported, and that I'm better off making my own. I won't be reopening this until I have my own interpretation coded.

**Comment by CRITAWAKETS** [comment]
In specific, i gave him advice that this version of pilk/peg nog wouldnt really fit how /tg/ does drinks.

**Comment by itseasytosee** [comment]
> And don't worry. Pilk and peg nog are still coming. Just in a different way of design.

God am I worried

File: code/datums/mood_events/drink_events.dm (hunk 2 above)
Comment (Jolly-66) [reply code/datums/mood_events/drink_events.dm:36]: I had the same reaction when I saw this too

And I fucking hate it 100%

**Pull request closed without merging**